src/lib/menu/menu-directive.ts

```diff
@@ -197,8 +197,8 @@ export class MatMenu implements OnInit, AfterContentInit, MatMenuPanel, OnDestro
   set classList(classes: string) { this.panelClass = classes; }
 
   /** Event emitted when the menu is closed. */
-  @Output() readonly closed: EventEmitter<void | 'click' | 'keydown'> =
-      new EventEmitter<void | 'click' | 'keydown'>();
+  @Output() readonly closed: EventEmitter<void | 'click' | 'keydown' | 'tab'> =
```

Comment:
Should we just make this type an enum now?

Reply:
I was trying to avoid breaking changes. I had another PR somewhere that changes it to an `Event` instead.